frontend/src/metabase/dashboard/components/ClickMappings.jsx

```diff
@@ -136,12 +137,9 @@ const ClickMappings = _.compose(
     }
 
     const [setTargets, unsetTargets] = _.partition(
-      getTargetsWithSourceFilters({
-        isAction: props.isAction,
-        isDash,
-        dashcard,
-        object,
-      }),
+      isDash
+        ? getTargetsForDashboard(object, dashcard)
+        : getTargetsForQuestion(object),
```

Comment:
The `getTargetsWithSourceFilters` function has been removed.
It used `isAction` and `isDash` options which are not really in TS spirit.